app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+
+    if len(command_list) == 3 or command.startswith("cp "):
+        _, file_name_in, file_name_out = command_list
+
+        if file_name_in != file_name_out:
+            try:
+                with (open(file_name_in) as file_in,
+                      open(file_name_out, "w") as file_out):
+                    file_out.write(file_in.read())
+            except (FileNotFoundError, PermissionError) as e:
+                raise f"Error {e}. The file does not exist or can not be read"
```

Comment:
`{e}` will already contain the error info, no need in additional `The file does not exist or can not be read` that can only confuse the user